docs/content/guides/dagster/code-references.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Code References in Dagster | Dagster Docs
+---
+
+# Linking to code in Dagster with Code References
+
+<Note>
+  This feature is considered <strong>experimental</strong> and is under active
+  development. This guide will be updated as we roll out new features.
+</Note>
+
+This guide explains how to use Code Reference metadata on your Dagster assets to link from the Dagster UI to the source code that produced the asset. In the local development environment, you can use this feature to navigate directly to the code backing an asset in your editor, and in the Dagster Cloud UI, you can link to the source code in your source control repository to see the full history of changes.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- A Dagster repository with assets that you want to link to code
+- Dagster version `1.7.6` or newer
+
+---
+
+## Referencing source code in your assets
+
+### Automatically attaching code references to asset definitions
```

Comment:
Because there's only one instructional section in this guide, I'd say we scrap `Referencing source code in your assets` and making the other two headings into H2s. If we plan to expand this to other definitions, though, maybe it's fine to keep it.